jena-arq/src/main/java/org/apache/jena/sparql/util/graph/GraphUtils.java

```diff
@@ -132,6 +133,15 @@ public static boolean atmostOneProperty(Resource r, Property p) {
         return true ;
     }
 
+    public static boolean getBooleanValue(Resource r, Property p) {
+        if ( !GraphUtils.atmostOneProperty(r, p) )
```

Comment:
Really nit-picking, and not related to this PR, but shouldn't that method be `atMost`? Like `atLeast`? Could be wrong, holidays minus :coffee: is not normally good for reviews.